editor/filesystem_dock.cpp

```diff
@@ -1064,6 +1064,13 @@ void FileSystemDock::_folder_option(int p_option) {
 				child = child->get_next();
 			}
 			break;
+		case FOLDER_SHOW_IN_EXPLORER:
+			String path = item->get_metadata(tree->get_selected_column());
+			String dir = GlobalConfig::get_singleton()->globalize_path(path);
+			dir = dir.substr(0, dir.find_last("/"));
```

Comment:
Not sure if this line is needed, since you can't be sure that `globalize_path` would always return the trailing `"/"`